dozer-cli/src/main.rs

```diff
@@ -312,27 +295,23 @@ fn parse_and_generate() -> Result<Cli, OrchestrationError> {
     )
 }
 
-fn init_configuration(cli: &Cli, runtime: Arc<Runtime>) -> Result<Config, CliError> {
-    dozer_tracing::init_telemetry_closure(
-        None,
-        &Default::default(),
-        || -> Result<Config, CliError> {
-            let res = runtime.block_on(init_config(
-                cli.config_paths.clone(),
-                cli.config_token.clone(),
-                cli.config_overrides.clone(),
-                cli.ignore_pipe,
-            ));
-
-            match res {
-                Ok(config) => {
-                    runtime.spawn(check_update());
-                    Ok(config)
-                }
-                Err(e) => Err(e),
+fn init_configuration(cli: &Cli, runtime: Arc<Runtime>) -> Result<(Config, Vec<String>), CliError> {
+    dozer_tracing::init_telemetry_closure(None, &Default::default(), || -> Result<_, CliError> {
+        let res = runtime.block_on(init_config(
+            cli.config_paths.clone(),
+            cli.config_token.clone(),
+            cli.config_overrides.clone(),
+            cli.ignore_pipe,
+        ));
+
+        match res {
```

Comment:
Can this be changed to a more readable block?
```
if let Ok(config) = res { 
  runtime.spawn(check_update());
}
res
```

Reply:
`check_update()` actually hasn't worked for a while, and in general is not very useful IMO, so I just removed it altogether.